js/freewheel.js

```diff
@@ -540,6 +541,34 @@ OO.Ads.manager(function(_, $) {
       // if (fwContext && _.isFunction(fwContext.dispose)) fwContext.dispose();
     };
 
+    var onResize = function() {
+      //Overlay placement issue - PBI-1227 as of 12/9/2015
+      //The main issue with Freewheel is when notifying their SDK of video size changes,
+      //Freewheel only attempts to resize ads and not re-position ads.
+      //In the Freewheel SDK, overlay ads do not have a resize function and thus nothing happens.
+
+      //We want to force the renderer to reposition the overlay ads. The Freewheel SDK has functions
+      //that accomplish this, but are undocumented.
+      if(currentPlayingSlot){
+        //These two calls update Freewheel with the new video sizes. At this point
+        //Freewheel will attempt to resize any ads
+        fwContext.setContentVideoElement(amc.ui.ooyalaVideoElement[0]);
+        fwContext.registerVideoDisplayBase(amc.ui.adWrapper.attr("id"));
```

Comment:
I'll add a function called notifySizeChange and call registerDisplayFor<>Ad in that function. That way we don't lose track of what we functionality we want to accomplish at this point.